spec/lib/workers/replication_worker_spec.rb

```diff
@@ -29,24 +29,21 @@
 
   context "testing child process heartbeat" do
     after do
+      Timecop.return
```

Comment:
I thought it wouldn't hurt to have it. But if it's not necessary at all I'll remove it.